### PR TITLE
lecture 6 task git api repos testing (rest-assured &apache httpclient) 

### PR DESCRIPTION
**Pull request by litvinyuk852**
lecture 6 task git api repos testing (rest-assured &apache httpclient)

### CODE REVIEW

File: Lecture6/src/test/java/GitReposUsersTestApache.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+public class GitReposUsersTestApache {
+    public static final String gitAPIUrl="https://api.github.com/";
+    public static final String gitReposOwner="litvinyuk852";
+    public static final String gitReposOwnerRepo="automationTasks";
+    public static final String gitAuthToken="f4b829d2f51cb382c6dc1d32511174c83c395d96";
+    public static final String gitInvitedUser="sannor";
+
+    @Test
+    public void printGitReposCollaboratorsApache() throws IOException {
+        String gitReposCollaborators = Request.Get(gitAPIUrl+"/repos/"+gitReposOwner+"/"+gitReposOwnerRepo+"/collaborators")
```
Comment (acherkas) [inline Lecture6/src/test/java/GitReposUsersTestApache.java]: rename gitReposCollaborators to CollaboratorsResponse

File: Lecture6/src/test/java/InvitationFailureCode.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    public String getFault() {
+        return fault;
+    }
+
+    public void setFault(String fault) {
+        this.fault = fault;
+    }
+
+    public String faultId;
```
[the remark this answers is not in the document]
Comment (litvinyuk852) [reply Lecture6/src/test/java/InvitationFailureCode.java]: The class is useless, is deleted. ResponseDto package with 5 new classes is created instead.

File: Lecture6/src/test/java/InvitationFailureCode.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    public String fault;
```
[the remark this answers is not in the document]
Comment (litvinyuk852) [reply Lecture6/src/test/java/InvitationFailureCode.java]: The class is useless, is deleted. ResponseDto package with 5 new classes is created instead.

File: Lecture6/src/test/java/InvitationSuccessCode.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public String message;
```
[the remark this answers is not in the document]
Comment (litvinyuk852) [reply Lecture6/src/test/java/InvitationSuccessCode.java]: The class is useless, is deleted. ResponseDto package with 5 new classes is created instead.

File: Lecture6/src/test/java/GitReposUsersTestApache.java (hunk 1 above)
Comment (litvinyuk852) [reply Lecture6/src/test/java/GitReposUsersTestApache.java]: renamed to gitReposCollaboratorsResponse

**Pull request merged**